archaius2-api/src/main/java/com/netflix/archaius/ConfigLoader.java

```diff
@@ -78,7 +76,7 @@
          * @param resourceName
          * @return CompositeConfig contains a full hierarchy of cascaded files
          */
-        CompositeConfig load(String resourceName) throws ConfigException;
+        Config load(String resourceName) throws ConfigException;
```

Comment:
This method returns CompositeConfig because configuration loading is inherently composite due to cascade loading.  Exposing CompositeConfig from the API makes that explicit and also allows for post processing of the response.  With that said, we would need to make CompositeConfig an interface for this to be part of the api package.